src/app/pages/applications/catalog/catalog.component.html

```diff
@@ -48,4 +48,5 @@ <h3 textLimiter threshold="15">{{ item.name }}</h3>
       </div>
     </div>
   </mat-card>
+  <entity-empty *ngIf="filteredCatalogApps.length == 0" [conf]="emptyPageConf"></entity-empty>
```

Comment:
Here and below == 0 is unnecessary

Reply:
I don't get it, you mean `*ngIf="filteredCatalogApps.length"` ?

Reply:
I mean that you can do !filteredCatalogApps.
I guess it's not too important.

Reply:
I don't think it will work here. ![] will give false.

Reply:
Sorry, I meant to write `!filteredCatalogApps.length`